downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc

```diff
@@ -50,8 +50,8 @@ When registration is complete, the registration page for the Automation Controll
 . In the *Automation controller application registration page* on Azure, copy and save the value of _Application (client) ID_.
 +
 You will use this value for the _Azure AD OAuth2 Key_ in the {PlatformNameShort} settings.
-. Under *Manage*, click *Certificates & secrets*.
-. Click *Client secrets* and then *+ New client secret*.
+. Under *Manage*, click btn:[Certificates & secrets].
+. Click btn:[Client secrets] and then btn:[+ New client secret].
```

Comment:
I'm not sure if these are actually buttons. I think Aine works on Azure doc, can you check with her?

Reply:
Hi @ariordan-redhat, are you able to let us know whether these elements (btn:[Client secrets] and btn:[+ New client secret]) are actually buttons? Thank you!

Reply:
I'll take a look

Reply:
@emurtoug They're not buttons.
![image](https://user-images.githubusercontent.com/44700011/214613651-db822023-166f-43d7-b1b3-34521542b15d.png)

Reply:
Thank you, @ariordan-redhat!

Reply:
> @emurtoug They're not buttons. ![image](https://user-images.githubusercontent.com/44700011/214613651-db822023-166f-43d7-b1b3-34521542b15d.png)

@dcdacosta I've changed this line to bold these elements instead: Click *Client secrets* and then *+ New client secret*.

Reply:
@emurtoug looks good. Thank you for the screen shot @ariordan-redhat, it really helps.